release/smbchk.py

```diff
@@ -70,12 +77,20 @@ def nm(ns):
     #   -C: demangles the C++ symbols.
     #   -fbsd: formats output in default 'bsd' style
     # note that we are only going to pick up symbols in the cyclus namespace
-    # because everything else should be external to the cares of cyclus stability
+    # because everything else should be external to the cares of cyclus
+    # stability
     lib = os.path.abspath(os.path.join(ns.prefix, 'lib', 'libcyclus.so'))
     stdout = subprocess.check_output(['nm', '-g', '-p', '-C', '-fbsd', lib])
     names = set()
-    ok_types = {'B', 'b', 'D', 'd', 'R', 'r', 'S', 's', 'T', 't', 'W', 'w', 'u'}
+    ok_types = {'B', 'b', 'D', 'd', 'R', 'r',
+                'S', 's', 'T', 't', 'W', 'w', 'u'}
     for line in stdout.splitlines():
+        # replace c++11 symbol by standart one
+        for symbol in cpp11_symbols:
+            line  =line.replace(symbol[0],symbol[1])
+        #fix string replament when a '>' follows the string
+        line = line.replace("string >", "string>")
```

Comment:
because those change affect only `string` and `list`:
https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html